src/property.js

```diff
@@ -0,0 +1,107 @@
+define([
+	'./object'
+], function(object) {
+	/**
+	 * A property represent an observable attribute of an object, with optional
+	 * getters and setters.
+	 *
+	 * @param{string} spec.object - Instance on which the property is installed.
+	 * @param{string} spec.name - Name of the property, which value of the same
+	 * name is installed on `my` on the instance.
+	 */
+	var property = object.subclass(function(that, my) {
+		my.initialize = function(spec) {
+			my.super(spec);
+			my.name = spec.name;
+			my.object = spec.object;
```

Comment:
as I previously said, `my.object` doesn't really gives an idea of what it does.

maybe `my.owner` instead or something similar?